features/facebookImport/src/views/infoScreens/activitiesInfoScreen/activitiesInfoScreen.jsx

```diff
@@ -8,7 +8,7 @@ const ActivitiesInfoScreen = () => {
     const activitiesInfoText = [
         <>
             <p>{i18n.t("activitiesInfoScreen:text1")}</p>
-            <div className="chart-description-title">
+            <div className="legend chart-description-title">
```

Comment:
Does "chart-description-title" *always* go with legend?

Reply:
chart-description-title has only these 3 use cases alongside with legend, yes. But I am not sure where and how legend is used elsewhere

Reply:
In that case, does `legend` cover all it's needed? or does that additional class provide something extra?